app/assets/v2/js/users.js

```diff
@@ -182,3 +198,14 @@ if (document.getElementById('gc-users-directory')) {
   });
 }
 
+if (getURLParams('invite')) {
+  let timer = setInterval(function() {
+    if ($('#reinvite-user').length) {
+      $('#reinvite-user').trigger('click');
+      setTimeout(function() {
+        $('#reinvite-user').trigger('click');
+      }, 200);
+      clearInterval(timer);
```

Comment:
Instead of this you can user the vm methods this.openBounties(reinviteUser), you don't need to go to trigger from the view.